ui-admin/src/study/participants/participantList/WithdrawnEnrolleeList.tsx

```diff
@@ -77,6 +79,12 @@ export default function WithdrawnEnrolleeList({ studyEnvContext }: { studyEnvCon
     getSortedRowModel: getSortedRowModel()
   })
   return <div className="container-fluid px-4 pt-4">
```

Comment:
spacing nit, this padding can be reduced now:
```suggestion
  return <div className="container-fluid px-4 pt-2">
```